### DISCUSSION TITLE
Inverter remote server B on datalogger to local server for monitoring?

### DISCUSSION THREAD

**Discussion by vizkoze** [hultenvp/solis-sensor#169] [Ideas]
Hello,
I recently installed a new PV system with a solis inverter and was looking for a way to integrate the data locally in my homeassistant install.
Yesterday, I received an email stating that API access tokens were disabled due to GPDR concerns and that access to the API would be restored in a future update. ETA unknown for the support guys.
So i read in this issuetracker that i could login into the logger (mine is an ethernet variant, not wifi) to view some basic info.
After searching for login and password (admin/admin) i noticed that the logger is able to connect to a second external server.
![image](https://user-images.githubusercontent.com/23431657/196730937-c966e563-9983-4d1b-b3c7-5208fd62865b.png)
Maybe it would be possible to listen to the output of that datastream if i could setup a local server. However I don't know what kind of server this would need to be as my programming skills are very basic.
Could someone point me in the right direction?

**Comment by hultenvp** [comment]
This is not the right place to ask as this is a discussion forum about the Solis cloud service integration, you might get better response on the HA discussion pages. Check out issue #162 and I'm sure you'll find an answer to your question and some further leads.
